src/js/components/DataTableColumns/__tests__/DataTableColumns-test.tsx

```diff
@@ -37,13 +38,40 @@ describe('DataTableColumns', () => {
     );
 
     expect(getByRole('button', { name: 'Open column selector' })).toBeTruthy();
-    expect(container.firstChild).toMatchSnapshot();
+    expect(asFragment()).toMatchSnapshot();
+
+    fireEvent.click(
+      screen.getByRole('button', { name: 'Open column selector' }),
+    );
+
+    window.scrollTo = jest.fn();
+
+    // advance timers so drop can open
+    act(() => jest.advanceTimersByTime(200));
+    // Take a snapshot of the tab panel
+    const firstTabPanel = screen.getByRole('tabpanel', {
+      name: 'Select columns Tab Contents',
+    });
+    expect(firstTabPanel).toMatchSnapshot();
+
+    // Click on the "Order columns" tab
+    fireEvent.click(screen.getByRole('tab', { name: 'Order columns' }));
+
+    act(() => jest.advanceTimersByTime(200));
```

Comment:
Do we need the advanceTimers again now that the drop is already open or can we reomve?
```suggestion
```

Reply:
let me verify